client/boot/common.js

```diff
@@ -124,7 +124,7 @@ const loggedInMiddleware = currentUser => {
 	}
 
 	page( '/', context => {
-		let redirectPath = '/read';
+		let redirectPath = 'variant' === abtest( 'redirectToCustomerHome' ) ? '/home' : '/read';
```

Comment:
For users who don't have a site, this will take them to the empty state:

![image](https://user-images.githubusercontent.com/14988353/72580864-79ed5880-3931-11ea-96d7-c18f734e1ae4.png)

What if we have the A/B test only act on users who have a primary site? We could also use the primary site to redirect them to the home of their primary site instead of the "select a site" screen, removing a step:

```suggestion
		const { primarySiteSlug } = currentUser.get(); 
		let redirectPath = primarySiteSlug && 'variant' === abtest( 'redirectToCustomerHome' ) ? `/home/${ primarySiteSlug }` : '/read';
```